apps/nerves_hub_www/lib/nerves_hub_www_web/controllers/product_controller.ex

```diff
@@ -15,7 +15,8 @@ defmodule NervesHubWWWWeb.ProductController do
 
   def index(%{assigns: %{user: user, org: org}} = conn, _params) do
     products = Products.get_products_by_user_and_org(user, org)
-    render(conn, "index.html", products: products)
+    changeset = Products.change_product(%Product{})
```

Comment:
Is this to support loading the new forms on the index screen? I think this will work as a stop gap but it won't allow the display of error messages since those are determined in the create action and then sent to the new screen. Unless that flow makes sense.

Reply:
Yeah this was just an initial attempt at getting the create form into a modal on that page. We might either want to not use all these form modals in the initial refresh or change to liveviews?

Reply:
Yeah this will get the form to display but it won't be totally functional. I would say to use individual pages for the different crud stuff where necessary for now unless Omar or co. think it makes sense to put in the effort to use liveviews here.

Reply:
Well the form DOES work, it's just a text input field. The product gets created though

Reply:
I'll remove for now though

Reply:
Right the create and stuff will work but the error stuff won't. Maybe that's fine. I don't think you can remove it without breaking that form.

Reply:
updated ^